apps/design/backend/src/store.ts

```diff
@@ -163,9 +165,18 @@ function hydrateElection(row: {
     sealUrl:
       rawElection.sealUrl ?? '/seals/state-of-hamilton-official-seal.svg',
   };
+
+  const electionData = JSON.stringify(election);
+
+  const electionDefinition: ElectionDefinition = {
+    election,
+    electionData,
+    electionHash: sha256(electionData),
+  };
```

Comment:
I thought about that a little bit, but was concerned that it was designed for tests. It might make sense to use `safeParseElectionDefinition` here. I think when I wrote this, I thought it wouldn't work due to having an incomplete object, but I think that actually might not be the case anymore (that was true in a previous iteration of this app).

Reply:
Ah ok yeah it's still an issue when creating a new election, since the current ElectionSchema validates that various items in the election are not empty (e.g. it requires at least one contest, ballot style, etc, and disallows empty strings).

I'm not sure if it makes sense to remove that validation or not...

Reply:
Got it - with all that added context, I think it's fine to leave as is